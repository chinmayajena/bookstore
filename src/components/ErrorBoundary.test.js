import React from "react";
import { shallow } from "enzyme";
import ErrorBoundary from "./ErrorBoundary";

describe("<ErrorBoundary />", () => {
  it("renders the component", () => {
    const component = shallow(<ErrorBoundary />);
    expect(component.debug()).toMatchSnapshot();
  });
});
