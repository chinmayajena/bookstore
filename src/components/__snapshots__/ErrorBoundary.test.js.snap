// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundary /> renders the component 1`] = `""`;
